dashboard/app/models/workshop.rb

```diff
@@ -9,9 +9,10 @@ class Workshop < ActiveRecord::Base
 
   has_many :teachers, through: :cohort, class_name: 'User'
 
-  # A Workshop is associated with a Cohort
-  belongs_to :cohort
-  has_many :districts, through: :cohort
```

Comment:
is the district association used anywhere? if it is, I think you might be able to `has_many :districts through :cohorts`